frontend/src/App.vue

```diff
@@ -10,16 +10,27 @@
     </v-content>
 
     <!-- footer -->
-    <v-footer v-if="$vuetify.breakpoint.mdAndUp"
+    <v-footer v-if="$vuetify.breakpoint.smAndUp"
               app color="grey lighten-5">
-      eCamp v0.1.0
+      <small>eCamp v0.1.0</small>
+      <v-spacer/>
+      <language-switcher />
     </v-footer>
   </v-app>
 </template>
 
 <script>
+import LanguageSwitcher from '@/components/layout/LanguageSwitcher'
 export default {
-  name: 'App'
+  name: 'App',
+  components: { LanguageSwitcher },
+  mounted () {
+    this.api.get()
+    const lang = 'de' // TODO save this value to database?
+    this.$root.$i18n.locale = lang
+    this.axios.defaults.headers.common['Accept-Language'] = lang
+    document.querySelector('html').setAttribute('lang', lang)
```

Comment:
Wenn du den LanguageSwitcher so definierst: `<language-switcher ref="languageSwitcher" />`, dann kannst du hier DRY schreiben:
```suggestion
    this.$refs.languageSwitcher.changeLang(lang)
```

Reply:
Kann das nicht ohne Refs gemacht werden. Dafür brauchts ja keinen Component. Evtl auslagern in i18n.js?